### PR TITLE
Chore/scarb

### PR DESCRIPTION
**Pull request by gaetbout**
Update the repo to use Scarb only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,25 +94,23 @@ use alexandria::math::sha512::sha512;
...
 ```bash
-make test
+scarb test -f math
 ```
 
 ### Format
 
 ```bash
-make format
+scarb format
```
Comment (maciektr) [inline README.md:113]: If you wish to format cairo files only:
```suggestion
scarb fmt
```

If you wish to use the script from Scarb manifest: 
```suggestion
scarb run format
```

Comment (maciektr) [inline README.md:113]:
```suggestion
scarb fmt
```

**Comment by maciektr** [comment]
This can be removed as well:
https://github.com/keep-starknet-strange/alexandria/blob/b92e41ea59de4881cfedff054d81b1738e54f0af/.github/workflows/test.yml#L5-L8

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
 Alexandria is a community maintained standard library for Cairo 1.0.
 It is a collection of useful algorithms and data structures implemented in Cairo.  
-Current version: [v2.0.0](https://github.com/starkware-libs/cairo/releases/tag/v2.0.0)
+Current version: [v2.0.1](https://github.com/starkware-libs/cairo/releases/tag/v2.0.0)
```
Comment (maciektr) [inline README.md:54]:
```suggestion
Current version: [v2.0.1](https://github.com/starkware-libs/cairo/releases/tag/v2.0.1)
```

**Review by maciektr — APPROVED** [review]

**Review by LucasLvy — APPROVED** [review]

**Pull request merged**